Move compute docs out (#516)
AlexsLemonade/OpenScPCA-analysis: docs/contributing-to-analyses/analysis-modules/compute-requirements.md

# Module compute requirements

When planning and implementing your analysis module, you will need to consider your module's **compute requirements.**
This refers to the computational resources, such as the number of CPUs or the amount of RAM, that are required for your analysis to run.

Knowing and reporting this information helps the OpenScPCA project organizers:

> - It helps us plan and allocate resources on Amazon Web Services (AWS) to run your module

Reviewer: ```suggestion
- It helps us plan and allocate resources on Amazon Web Services (AWS) to run your module.
```

> - It provides the compute context we need while [reviewing your pull requests](../pr-review-and-merge/index.md) (e.g., can we review using our laptops vs. using AWS).

Reviewer: "provides the compute context we need" seemed unclear

```suggestion
- It lets us know what compute resources we need while [reviewing your pull requests](../pr-review-and-merge/index.md) (e.g., can we review using our laptops vs. using AWS).
```
